app/decorators/litigation_side_decorator.rb

```diff
@@ -0,0 +1,11 @@
+class LitigationSideDecorator < Draper::Decorator
```

Comment:
this class - `LitigationSideDecorator` isn't used anywhere, is it?

Reply:
ahh, draper magic, it is used [here](https://github.com/Vizzuality/laws_and_pathways/blob/feature/litigations/app/admin/litigations.rb#L44) maybe we should do it more explicit.